introduction/java_loops_i/Solution.java

```diff
@@ -0,0 +1,14 @@
+import java.util.Scanner;
+
+public class Solution {
+  static private final int MAX_NUMBER_OPERATIONS = 10;
+  public static void main(String[] args) {
+    Scanner input = new Scanner(System.in);
+    int inputNumber = input.nextInt();
+    int result;
+    for (int i = 1; i <= MAX_NUMBER_OPERATIONS; i++) {
+      result = inputNumber * i;
```

Comment:
if you use it only here - declare it here. Do not use scope broader than you need. In this case the scope of using result variable is for-loop, while declaration scope is the whole main function.